Add additional CanvasRenderingContext2D.textBaseline example

fuchunhui opened the pull request:
<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->
enrich example of CanvasRenderingContext2D.textBaseline.

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->
make it easier to understand the differences between baselines.

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->

#### Related issues
<!-- 🔨 If applicable, use "Fixes #XYZ" -->
Fixes #14495 

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

fuchunhui committed:
Add an example on  (#14495)

sideshowbarker committed:
Avoid making user scroll horizontally; update intro paragraph

sideshowbarker commented:
Thanks much, and congrats on landing your first docs change here — welcome aboard 🎉

I added an additional commit to change the layout slightly in order to avoid making the user need to scroll horizontally when looking at the example — and also updated the intro text to make more clear why we’re providing two examples.

fuchunhui commented:
> Thanks much, and congrats on landing your first docs change here — welcome aboard 🎉
> 
> I added an additional commit to change the layout slightly in order to avoid making the user need to scroll horizontally when looking at the example — and also updated the intro text to make more clear why we’re providing two examples.

Thanks, I get more.
